public/index.html

```diff
@@ -21,12 +21,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <!-- Google alef font. see https://fonts.google.com/specimen/Alef -->
-    <link
-      href="https://fonts.googleapis.com/css2?family=Alef:wght@400;700&display=swap"
-      rel="stylesheet"
-      crossorigin="anonymous"
-    />
+    <!-- Google Rubik font. see https://fonts.google.com/specimen/Rubik -->
+    <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet" crossorigin="anonymous" />
```

Comment:
we probably wish to have the font also in bold-700 as I see in the old font

Reply:
I asked yuval about it , he said to wait for the designer who assigned to this issue to tell me how exactly how he want it to be  . meanwhile i put the regular font .

Reply:
I offer to add also the bold one